docs/developer/v1alpha1-compared-to-v1alpha2.md

```diff
@@ -0,0 +1,65 @@
+# Cluster API v1alpha1 compared to v1alpha2
+
+## Providers
+
+### v1alpha1
+
+Providers in v1alpha1 are generic cloud providers including, aws, gcp, openstack, azure, vsphere and others. The provider 
+vendors in Cluster API's controllers, registers their actuators with the controllers and runs a single manager 
+(with the Cluster API controllers) to complete a Cluster API management cluster.
+
+### v1alpha2
+
+v1alpha2 introduces new providers. Cluster API's controllers are no longer vendored by providers. Instead, Cluster API 
```

Comment:
I'd probably move this section at the top:
v1alpha introduces 2 new providers (infrastructure and bootstrap provider). This means that in order to have a complete management cluster that is ready to build clusters you will need three managers: infrastructure, bootstrap, and cluster api. 

Everything else ->

Reply:
I organized it this way so i could, briefly, define what a provider is. That allows me to make a comparison in this section. Without the definition in v1a1 I was thinking this might be a little confusing.

What are your thoughts about why you think this section should move to the top?

Reply:
*To the top of v1alpha2 section. Sorry I wasn't clear.
As for why to the top of v1alpha2, it just sets up the section's context better so that I have an overview.

Reply:
restructured, thanks for the feedback